DEVELOPMENT.md

```diff
@@ -2,48 +2,28 @@
 
 - [Developer's Guide](#developers-guide)
   - [Setting Up Your macOS Development Environment](#setting-up-your-macos-development-environment)
-  - [Running Scripts with `hatch`](#running-scripts-with-hatch)
+  - [Testing and Linting](#testing-and-linting)
   - [Installing Pre-Commit Hooks](#installing-pre-commit-hooks)
-  - [Building the `phoenix` Package](#building-the-phoenix-package)
-  - [Installing a `phoenix` Build](#installing-a-phoenix-build)
+  - [Building the Package](#building-the-package)
+  - [Installing a Phoenix Build](#installing-a-phoenix-build)
+  - [Installing a `git` Branch on Colab](#installing-a-git-branch-on-colab)
   - [Setting Up Your Windows Test Environment](#setting-up-your-windows-test-environment)
     - [Selecting a Virtualization Option](#selecting-a-virtualization-option)
-    - [Installing Python and `phoenix`](#installing-python-and-phoenix)
+    - [Installing Python and Phoenix](#installing-python-and-phoenix)
     - [Configuring a Remote Interpreter](#configuring-a-remote-interpreter)
     - [Troubleshooting](#troubleshooting)
   - [Publishing a New Release](#publishing-a-new-release)
```

Comment:
did you want to outline your integration-tests and how to run them>

Reply:
Going to punt on this one, probably needs a new `hatch` command.